src/main/scala/com/github/unchama/seichiassist/listener/PlayerInventoryListener.scala

```diff
@@ -613,5 +614,55 @@ class PlayerInventoryListener(implicit effectEnvironment: EffectEnvironment,
     } else if (topinventory.getTitle == LIGHT_PURPLE.toString + "" + BOLD + "スキルを進化させました") {
       event.setCancelled(true)
     }
+
+  }
+
+
+  @EventHandler
+  def onItemNameRemoveEvent(event: InventoryCloseEvent): Unit = {
+    val player = event.getPlayer.asInstanceOf[Player]
+  //エラー分岐
+    val inventory = event.getInventory
+
+    //インベントリサイズが36でなければ処理を終了させる
+    if (inventory.row != 4){
+      return
+    }
+    if (inventory.getTitle == s"${GOLD}${BOLD}名義をなくしたいアイテムを投入してください") {
+      val item = inventory.getContents
+
+      var count = 0
+      //for文を使い、1つずつアイテムを見ていく
+      for(m <- item){
+        if(m !=null){
+          if (m.getItemMeta.hasLore){
+            val itemstack: ItemStack = m.getData.toItemStack
+            if (Util.itemStackContainsOwnerName(itemstack:ItemStack, player.getName)){
```

Comment:
```suggestion
            if (Util.itemStackContainsOwnerName(itemstack, player.getName)){
```

`itemstack:ItemStack`における`ItemStack`は「[みなし型](http://yanana.github.io/scala-style/types/ascription.html)」ですが、639行目の宣言から自明であるため不必要です。

Reply:
これも治っていません